projects/organization-management/src/app/models/b2b-user/b2b-user.mapper.ts

```diff
@@ -17,6 +17,7 @@ export class B2bUserMapper {
         firstName: AttributeHelper.getAttributeValueByAttributeName(e.attributes, 'firstName'),
         lastName: AttributeHelper.getAttributeValueByAttributeName(e.attributes, 'lastName'),
         roleIDs: AttributeHelper.getAttributeValueByAttributeName(e.attributes, 'roleIDs'),
+        // ToDo: #IS-31051: map the active flag
```

Comment:
You can already insert the code and remove the TODO. It will just start working, when the fix arrives 😄 
But, please add a note as "known limitation" in the PR description.